LICENSE-binary

```diff
@@ -0,0 +1,289 @@
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+------------------------------------------------------------------------------------
+This product bundles various third-party components under other open source licenses.
+This section summarizes those components and their licenses. See licenses/
+for text of these licenses.
+
+Apache License Version 2.0
+--------------------------
+org.roaringbitmap:RoaringBitmap
+org.apache.commons:commons-crypto
+commons-io:commons-io
+org.apache.commons:commons-lang3
+commons-logging:commons-logging
+com.google.guava:guava
+org.apache.hadoop:hadoop-client-api
+org.apache.hadoop:hadoop-client-runtime
+org.apache.htrace:htrace-core4
+org.slf4j:jcl-over-slf4j
+com.google.code.findbugs:jsr305
+org.apache.logging.log4j:log4j-1.2-api
+org.apache.logging.log4j:log4j-api
+org.apache.logging.log4j:log4j-core
+org.apache.logging.log4j:log4j-slf4j-impl
+io.dropwizard.metrics:metrics-core
+io.dropwizard.metrics:metrics-graphite
+io.dropwizard.metrics:metrics-jvm
+io.netty:netty-all
+io.netty:netty-buffer
+io.netty:netty-codec
+io.netty:netty-codec-dns
+io.netty:netty-codec-haproxy
+io.netty:netty-codec-http
+io.netty:netty-codec-http2
+io.netty:netty-codec-memcache
+io.netty:netty-codec-mqtt
+io.netty:netty-codec-redis
+io.netty:netty-codec-smtp
+io.netty:netty-codec-socks
+io.netty:netty-codec-stomp
+io.netty:netty-codec-xml
+io.netty:netty-common
+io.netty:netty-handler
+io.netty:netty-handler-proxy
+io.netty:netty-resolver
+io.netty:netty-resolver-dns
+io.netty:netty-resolver-dns-classes-macos
+io.netty:netty-resolver-dns-native-macos
+io.netty:netty-transport
+io.netty:netty-transport-classes-epoll
+io.netty:netty-transport-classes-kqueue
+io.netty:netty-transport-native-epoll
+io.netty:netty-transport-native-kqueue
+io.netty:netty-transport-native-unix-common
+io.netty:netty-transport-rxtx
+io.netty:netty-transport-sctp
+io.netty:netty-transport-udt
+org.apache.ratis:ratis-client
+org.apache.ratis:ratis-common
+org.apache.ratis:ratis-grpc
+org.apache.ratis:ratis-metrics
+org.apache.ratis:ratis-netty
+org.apache.ratis:ratis-proto
+org.apache.ratis:ratis-server
+org.apache.ratis:ratis-server-api
+org.apache.ratis:ratis-thirdparty-misc
+org.scala-lang:scala-library
+org.scala-lang:scala-reflect
+org.roaringbitmap:shims
```

Comment:
how do you sort this file? look `org.roaringbitmap:*` appear on the first and nearly last